chess/__init__.py

```diff
@@ -1221,14 +1221,19 @@ def __str__(self) -> str:
 
         return "".join(builder)
 
-    def unicode(self, *, invert_color: bool = False, borders: bool = False, empty_square: str = "⭘") -> str:
+    def unicode(self, *, invert_color: bool = False, borders: bool = False, empty_square: str = "⭘", flipped:bool = False) -> str:
```

Comment:
`orientation` is preferable in `chess.svg`. Let's use it here, too.

https://github.com/niklasf/python-chess/blob/aa0a04f37da9339b7d3a983aecbe9c97935fc0d3/chess/svg.py#L292-L293